Homework_4/lesson_4.6.py

```diff
@@ -0,0 +1,17 @@
+# ✔ Функция получает на вход список чисел и два индекса.
+# ✔ Вернуть сумму чисел между переданными индексами.
+# ✔ Если индекс выходит за пределы списка, сумма считается
+# до конца и/или начала списка.
+
+def my_func(arr, start, stop):
+    if stop > len(arr):
+        stop = len(arr)
+    if start < len(arr):
+        start = arr[0]
+
+        # [start+1, stop])
+    return sum(arr[(start + 1):(stop - 1)])
+
+
+arr = [1, 2, 3, 4, 5, 6, 7]
+print(my_func(arr, -10, 100))
```

Comment:
В условии не сказано, как нужно задавать индексы, то есть можно как угодно. Я задал так:
```suggestion
arr = [1, 2, 3, 4, 5, 6, 7]
print(my_func(arr, 5, 2))
```
И получил такой ответ
![image](https://github.com/MaxiSound/PYTHON_IMPROVED_HOMEWORKS/assets/49741194/64c9e2a8-2941-4c9a-a9c4-1b687b77e5e9)